pkg/test/resourcefixture/testdata/basic/discoveryengine/v1alpha1/discoveryenginedatastore/discoveryenginedatastoreminimal/_http.log

```diff
@@ -56,8 +56,7 @@ X-Xss-Protection: 0
     "defaultSchemaId": "default_schema",
     "displayName": "My first data store",
     "industryVertical": 1,
-    "name": "projects/${projectNumber}/locations/global/collections/default_collection/dataStores/datastore-${uniqueId}",
-    "servingConfigDataStore": {}
```

Comment:
I think we want this `servingConfigDataStore` field showing in the GCP object.

Reply:
It's not in the protos yet